README.md

```diff
@@ -241,6 +241,7 @@ Currently **officially** using Amundsen:
 1. [Square](https://squareup.com/us/en)
 1. [WeTransfer](https://wetransfer.com)
 1. [Workday](https://www.workday.com/en-us/homepage.html)
+1. [Convoy](https://www.convoy.com/)
```

Comment:
thanks @lexisantoro ! could you : 
1. sign the pr (git commit --amend --signoff)
2. put it in the sort order?

Thanks for the quick pr!